cmd/wasmcli/main.go

```diff
@@ -39,7 +39,6 @@ func main() {
 	config.SetBech32PrefixForAccount(sdk.Bech32PrefixAccAddr, sdk.Bech32PrefixAccPub)
 	config.SetBech32PrefixForValidator(sdk.Bech32PrefixValAddr, sdk.Bech32PrefixValPub)
 	config.SetBech32PrefixForConsensusNode(sdk.Bech32PrefixConsAddr, sdk.Bech32PrefixConsPub)
-	config.SetKeyringServiceName("wasmd")
```

Comment:
Interesting that you removed this, but I guess this functionality got removed from the sdk. Huh.

I tried it out and it does use a different keyring than before. But now it is called `wasm` rather than `wasmd`. Wondering where the name comes from. Guess this is not your work, just curious.

Thanks for making the fix.

Reply:
It comes from the App Name I believe